dev/release/verify-release-candidate.sh

```diff
@@ -484,7 +484,7 @@ test_glib() {
 
   # Install bundler if doesn't exist
   if ! bundle --version; then
-    gem install --user-install --no-document bundler
+    gem install --no-document bundler --install-dir="${ARROW_TMPDIR}/glib-build/gems" pkg/*.gem -- ${gem_flags}
```

Comment:
If we do this, don't we also need to add this directory to $PATH?

Reply:
Need to check this. It would take a day or two. Hopefully that is ok?

Reply:
No rush, 0.2.0 isn't happening for a while :)